src/annotator/sidebar.js

```diff
@@ -67,6 +69,22 @@ export default class Sidebar {
   constructor(element, eventBus, guest, config = {}) {
     this._emitter = eventBus.createEmitter();
 
+    /**
+     * Tracks which `Guest` has a text selection. It uses the frame identifier
+     * which uniquely identifies each `Guest`. If there is no text selected, the
+     * value is set to `null`.
+     *
+     * @type {null|string}
+     */
+    this._selectionAt = null;
```

Comment:
```suggestion
    this._guestWithSelection = null;
```

This would make it more immediately obvious what this field is.